### DISCUSSION TITLE
Anyone really good at setting up a self-host server?

### DISCUSSION THREAD

**Discussion by dhcpserver** [simple-login/app#538] [Self-hosting question]
I didn't realize how dependent I've been on the simplelogin.io site until it's been mostly down for the last 24 hours and all my domains aren't forwarding mail.  Some of my accounts have two-factor authentication and since it's taking 8+ hours for mail to forward, I'm having problems doing important things, like paying my rent because the link is timing out before I get it.

I've built things like DNS servers using Droplets before, but I'm struggling trying to get SimpleMail set up as a self-hosted service.  End goal is that I want a standalone service just for my use so I have nobody to blame but myself if it goes down.

Is there someone willing to build it out for me (I'll pay some something if it's affordable) if I supply the droplet with a box you can drop into and configure?  I know my way around DNS and can take care of all those things, I just need someone who really knows their way around knocking these out as a standalone installation that can do the server part.

Drop a note if this is you and want some easy beer money.

Thanks!

**Comment by nguyenkims** [comment]
Hi, this downtime incident is unacceptable and on behalf of the team, I deeply apologize for any inconvenience it has caused. Unfortunately hosting email isn't easy and even popular email services can experience several hours of downtime. We hope we have learned a lot via this bad experience and can bring back the stability to SimpleLogin.

**Reply by dhcpserver** [reply]
Thank you for your reply, Son.  The service seems to be working 100% and quickly now. Thank you for addressing it quickly.

Can you share the problem cause and what was done to prevent it in the future?

Thanks, Son.

**Reply by nguyenkims** [reply]
Yes we'll write a blog post about what happened, the solution, and how to prevent it from happening in the future.

**Reply by dhcpserver** [reply]
Great blog post and appreciate sharing all the lessons learned.  Thanks for sharing the root cause with the community!
